src/components/app.js

```diff
@@ -13,7 +13,8 @@ import { text } from 'd3-request';
 
 class App extends React.Component {
 
-  componentDidMount() {
+  async componentDidMount() {
+    await new Promise(resolve => setTimeout(resolve, 1000));
```

Comment:
I really don't understand the lifecycle. Go ahead and try it and see what happens :-)